README.md

```diff
@@ -170,11 +170,16 @@ class InitCommand extends Ahc\Cli\Input\Command
     {
         parent::__construct('init', 'Init something');
 
+        $help = '<cyan>Custom help screen</end>';
+        $writer = new Ahc\Cli\Output\Writer();
+        $rendered_help = $writer->colorizer()->colors($help);
```

Comment:
$camelCase or it can be just done below: `->help($writer->colorizer()->colors($help))`

Reply:
Updated!